keyboards/chili/README.md

```diff
@@ -0,0 +1,15 @@
+# YDKB Chili
+
+[Chili PCB](https://gd3.alicdn.com/imgextra/i3/12420845/O1CN01ilPFVn1I73GjalIXf_!!12420845.jpg)
```

Comment:
This image doesn't seem to be loading in the Markdown preview, could you rehost it on imgur?